package.json

```diff
@@ -19,7 +19,9 @@
 	"sideEffects": false,
 	"scripts": {
 		"start": "start-storybook -p 6006",
-		"build": "rollup -c && tsc -p ./tsconfig.json && npm run doc",
+		"copy-manifest": "cp pkg-json-manifest dist/package.json",
```

Comment:
It seems like `copy-manifest` is defined here but never used? If that is the case could we make sure to remove it before merging this PR?

Reply:
This is what I wanted to discuss with you, as this script has to be run at publish time in order to create the subdirectories for the `/react` imports

Reply:
Resolved by 2b0eb922b5eff7d077c023265d52308af1554454